FSI/rigid_flap/OpenFOAM-RigidBody/Solid/rigid_body_solver.cpp

```diff
@@ -0,0 +1,246 @@
+#include <iostream>
+#include <math.h>
```

Comment:
Better use the C++ headers:
```suggestion
#include <cmath>
```